services/s3/src/main/resources/codegen-resources/customization.config

```diff
@@ -172,5 +172,14 @@
       "clientConfiguration"
     ]
   },
+  "additionalBuilderMethods": [
+      {
+        "methodName": "crtClientBuilder",
+        "delegateBuilder": "builder",
+        "javaDoc": "{@link S3CrtAsyncClientBuilder} to create instance of {@link S3CrtAsyncClient}",
```

Comment:
Do we need javadoc field? Can we construct the Javadoc based on existing information? We can have something similar to the javadocs for the standard builder method.
```
 Create a builder that can be used to configure and create a {@link S3CrtAsyncClient}
```

Reply:
done